pkg/mkverification-iso-efi/make-efi

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+#
+# input is a tarball on stdin with kernel and cmdline in /boot
+# output is an iso on stdout
+#
+# The following env variables change the behaviour of this script
+#     DEBUG - makes this script verbose
+
+set -e
+[ -n "$DEBUG" ] && set -x
+
+copy() {
+   dd if="$1" of="$2" bs=1M 2>/dev/null
+}
+
+#
+# Extract partitions from stdin or look them up in /bits
+#
+mkdir /parts 2>/dev/null || :
+cd /parts
+if [ -d /bits ]; then
+   ln -s /bits/* .
+   rm \* >/dev/null 2>&1 || :
+else
+   bsdtar xzf -
+fi
+
+# for verification we need to add a uuid and override grub.cfg
+if [ "$1" = verification ]; then
+   mkdir -p boot && touch boot/.boot_repository
+   od -An -x -N 16 /dev/random | tr -d ' ' > boot/.uuid
+   cp grub.cfg EFI/BOOT/
+fi
+
+# create a ISO with a EFI boot partition
+# Stuff it into a FAT filesystem, making it as small as possible.  511KiB
+# headroom seems to be enough; (x+31)/32*32 rounds up to multiple of 32.
+mkfs.vfat -v -C boot.img \
+        $(( ($(stat -c %s EFI/BOOT/BOOT*EFI) / 1024 + 511) \
+        / 32 * 32 )) > /dev/null
+
+mmd -i boot.img ::/EFI
+mmd -i boot.img ::/EFI/BOOT
+mcopy -i boot.img EFI/BOOT/* ::/EFI/BOOT/
+
+# pay attentio to the -f option: it follows symlinks which is exactly
+# what we need for installer construction but could be dangerous in the
+# generic case for this container
+xorriso -as mkisofs \
+        -m rootfs-\* -f -R -e boot.img -hide boot.img -hide boot.catalog -no-emul-boot -o /tmp/disk.iso .
+copy /tmp/disk.iso /output.iso
+
+# How to build a VHDX. Commented out because we are currently not using it
```

Comment:
This comment is already part of pkg/mkimage-iso-efi/make-efi (from where you took the code), I don't think we need to keep a copy here, you can safely remove it....